src/aiida_quantumespresso/workflows/protocols/xspectra/crystal.yaml

```diff
@@ -0,0 +1,14 @@
+default_inputs:
+    abs_atom_marker: X
+    clean_workdir: True
+    return_all_powder_spectra: False
+    core:
+        get_powder_spectrum: True
+default_protocol: moderate
+protocols:
```

Comment:
Increasing `supercell_min_parameter` beyond the required amount has almost no effect on the accuracy of the calculation. The reason for the supercell is to prevent a spurious interaction between neighbouring core-hole images, since we are localising a core-hole in a periodic structure, so the periodic cell parameter will either be too small to separate the images or large enough that the spurious interaction becomes negligible.

In testing, we find that 8.0 ang is sufficient for this - both for metals and insulators, in agreement with the [original XSpectra publication](dx.doi.org/10.1103/PhysRevB.66.195107) - though this can be higher depending on both the system type and the choice of core-hole approximation. For instance, when studying a carbon anode material more recently, we found that for a system of graphene + Li the cell size convergence is very slow using a full-core-hole approach but agrees with the minimum parameter of 8.0 using an excited-core-hole approach.

The only other parameters which affect the accuracy of the XAS spectrum are already covered by the various other `WorkChain`s called in the workflow, so the protocol options in the `CrystalWorkChain` are really just there to set the level of accuracy in the sub-processes.

Reply:
The parameter in the protocol should be well-tested so that the user can use it for safety. I suggest that we do some testing on the supercell size. And show the results in one of our meetings with others. Of course, no need to do this for merging this PR.